Tasks/NuGetToolInstallerV1/make.json

```diff
@@ -14,7 +14,7 @@
     "rm": [
         {
             "items": [
-                "node_modules/packaging-common/node_modules/vsts-task-lib"
+                "node_modules/azure-pipelines-packaging-common/node_modules/azure-pipelines-task-lib"
```

Comment:
is this really needed?  What happens if we don't include it in the rm section?

Reply:
The above change is required as vsts-task-lib and packaging-common both are deprecated and thus needs to be replaced with the new packages. Since these packages  are no longer maintained, they haven't been migrated to Node16 and thus there were build error when using these packages.
You can also find a similar explanation in the modified Node16 migration guide.
![image](https://user-images.githubusercontent.com/117078846/206538121-97c2b797-c7be-43f5-9b48-6c3573f6ae3a.png)